.github/workflows/ci.yml

```
@@ -484,3 +484,133 @@ jobs:
         run: yarn up eslint@7.5.0
       - name: Run babel/eslint tests
         run: node ./node_modules/.bin/jest eslint
+
+  test262-prepare:
+    name: test262 - Prepare for running in parallel
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Babel
+        uses: actions/checkout@v3
+      - name: Use Node.js latest
+        uses: actions/setup-node@v3
+      - name: Checkout test runner
+        uses: actions/checkout@v3
+        with:
+          repository: babel/babel-test262-runner
+          path: babel-test262-runner
+      - name: Download test262
+        run: make bootstrap-test262
+      - name: Prepare test262 chunks
+        run: |
+          cd babel-test262-runner
+          npm ci
+          node lib/prepare-chunks 8 > ~/test262-chunks.json
+        env:
+          TEST262_PATH: ../build/test262
+      - name: Upload chunks artifact
+        uses: actions/upload-artifact@v3
+        with:
+          name: test262-chunks
+          path: ~/test262-chunks.json
+          retention-days: 3
+
+  test262:
+    name: test262
+    strategy:
+      matrix:
+        chunk: [0, 1, 2, 3, 4, 5, 6, 7]
+    runs-on: ubuntu-latest
```

```suggestion
    runs-on: macos-latest
```

@liuxingbaoyu we can try